docs/Administrator-Guide/Setting-Up-Clients.md

```diff
@@ -50,9 +50,13 @@ To install Gluster Native Client on RPM distribution-based systems
 2.  Ensure that TCP and UDP ports 24007 and 24008 are open on all
     Gluster servers. Apart from these ports, you need to open one port
     for each brick starting from port 49152 (instead of 24009 onwards as
-    with previous releases). The brick ports assignment scheme is now
-    compliant with IANA guidelines. For example: if you have
-    five bricks, you need to have ports 49152 to 49156 open.
+    with previous releases). The brick ports are now randomized, a port is
+    randomly selected within the range of base_port to max_port as defined
+    in glusterd.vol file and then assigned to the brick. For example: if you have
+    five bricks, you need to have at least 5 ports open within the given range of
+    base_port and max_port.
+    To reduce the number of open ports, one can lower the max_port value in the
```

Comment:
It may be appropriate to suggest why reducing the number of open ports is being recommended - security best practices I am presuming.

Reply:
Done